### PR TITLE
CDAP-5903 Add info about SFTP also being supported in FTP batch source

### PR DESCRIPTION
**Pull request by gokulavasan**
JIRA : https://issues.cask.co/browse/CDAP-5903

### CODE REVIEW

**Comment by gokulavasan** [comment]
@sreevatsanraman @awholegunch Please review when you get a chance. Thanks!

File: core-plugins/docs/FTP-batchsource.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# FTP/SFTP Batch Source
```
Comment (awholegunch) [inline core-plugins/docs/FTP-batchsource.md]: Putting a slash in the name could be problematic...what about a hyphen?

```
# FTP-SFTP Batch Source
```

Comment (awholegunch) [inline core-plugins/docs/FTP-batchsource.md]: An alternative would be to call it an `SFTP Batch Source`, where FTP is just one of the other protocols it handles.

File: core-plugins/widgets/FTP-batchsource.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   },
   "configuration-groups": [
     {
-      "label": "FTP Batch Source Properties",
+      "label": "FTP/SFTP Batch Source Properties",
```
Comment (awholegunch) [inline core-plugins/widgets/FTP-batchsource.json]: Perhaps

```
"label": "FTP-SFTP Batch Source Properties",
```

File: core-plugins/docs/FTP-batchsource.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+**inputFormatClass:** Name of the input format class, which must be a
+subclass of FileInputFormat. Defaults to CombineTextInputFormat.
+
+
+Example
+-------
+This example connects to an SFTP server and reads in files found in the specified directory.
+
+    {
+        "name": "FTP",
```
Comment (awholegunch) [inline core-plugins/docs/FTP-batchsource.md:35]: `SFTP`? `FTP-SFTP`?

**Comment by awholegunch** [comment]
I have edited the documentation, but left some comments about naming. Please check my commits, but aside from the naming issue, the docs LGTM.

File: core-plugins/docs/FTP-batchsource.md (hunk 1 above)
Comment (gokulavasan) [inline core-plugins/docs/FTP-batchsource.md]: I would also prefer calling it SFTP Batch Source. I think the number of use cases for SFTP might be greater than FTP source. @sreevatsanraman Any thoughts?

File: core-plugins/docs/FTP-batchsource.md (hunk 3 above)
Comment (gokulavasan) [inline core-plugins/docs/FTP-batchsource.md:35]: We are leaving it as FTP

**Comment by gokulavasan** [comment]
@sreevatsanraman @awholegunch Can I get a final review on this PR? I will squash and merge the changes.

**Comment by sreevatsanraman** [comment]
lgtm

**Comment by awholegunch** [comment]
Made minor change, please check. Otherwise, LGTM.

**Pull request merged**